sdks/java/core/src/main/java/org/apache/beam/sdk/annotations/Experimental.java

```diff
@@ -76,6 +76,9 @@
     TIMERS,
 
     /** Experimental APIs related to customizing the output time for computed values. */
-    OUTPUT_TIME
+    OUTPUT_TIME,
+
+    /** Metrics-related experimental APIs. */
+    METRICS
```

Comment:
Somewhat arbitrary, although we've tended to mark things as experimental if they are likely to change and/or not completely stable.